src/ManTradesTable.js

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import TradeRow from './TradeRow'
+import { PieChart } from 'react-minimal-pie-chart';
```

Comment:
you'll have to commit the package.json file as well so the build server knows to reference